### ISSUE TITLE
Unexpected behaviour when Diff and Trim are specified in the same mix

### ISSUE THREAD

**Issue by elecpower** [EdgeTX/edgetx#1244]
This mainly affects fixed wing models though from the very long discussion there are other use cases.
This was raised in OpenTX but never resolved https://github.com/opentx/opentx/issues/503
A solution was put forward but never made it https://github.com/opentx/opentx/pull/2361
Reference https://rc-soar.com/opentx/setups/diff/diff_bug.htm
I support the enhancement so I'll put if forward for the EdgeTX brain trust to consider.

**Comment by eshifri** [comment]
I always thought it was a feature - not a bug. :-)
If I want the "correct" trim behavior, I can add trim as a separate line in a mix with weight, let's say 0.25 and disable trim on the stick.

**Comment by elecpower** [comment]
As it currently operates, its a gotcha for a pilot who is not aware of this 'feature' especially since it is not formally documented.

One complaint about using ETX/OTX is that its too complex for newbies. Also some pilots would be happy with templates and wizards to set up and maintain their models without getting into the 'weeds'.

Another situation is where you have that brand new glider and it flies straight and true so no need for splitting the mix lines. Then after a few mishaps it needs a tweak to the trims, even after hardware adjustment. Further more we have the copy trims to sub-trims function. At that point the 'feature' raises its head. After some searching forums and issues you may stumble upon the fix.

**Comment by raphaelcoeffic** [comment]
This is a quite long and historical discussion on that topic! 😳

I would suggest modeling the different options so we are clear on what we want to have at the end.

**Comment by eshifri** [comment]
I agree that current behavior is counterintuitive and the examples/models provided in OTX discussions are rather convincing.
The concern is the different behavior in OTX and ETX. That being said, if it is documented and is an improvement...

**Comment by gagarinlg** [comment]
If I understand that correctly, the trim is applied to the stick and then the sum is is put into diff.
I think this resembles the behavior of old radios, where the trim rotated the potentiometer. In think this is a reasonable way to implement this.
Did I miss something?

**Comment by eshifri** [comment]
@gagarinlg There was a long discussion here:  opentx/opentx#503
One cam apply the same argument to the expo, right? But in the case of expo we definitely want to exclude trim.
In some sense, differential is similar: you wan to have different sensitivity to the stick in opposite directions.
So one can find argument pro and contra both ways of doing it.
With the current behavior, if one wants to have a proper differential ailerons, he has to use trim as a setup line in the mix. Not a big deal IMO, if the behavior is known and understood. Unfortunately, many pilots find it out by learning on their own mistakes.

**Comment by philmoz** [comment]
After reading https://rc-soar.com/opentx/setups/diff/diff_bug.htm I did some testing.

While disabling trims in the mixer and adding new mix lines to add the trim back in works, the same result can be achieved by using two inputs and setting the Diff value on the inputs instead of the mix.

I may be missing something; but this appears to work:
<img width="419" height="74" alt="Image" src="https://github.com/user-attachments/assets/742a45c6-bfda-4864-aa56-d34a25baa195" />
<img width="389" height="73" alt="Image" src="https://github.com/user-attachments/assets/02cda247-b901-4e05-8b9d-c8e8f8c0bdec" />

**Comment by philmoz** [comment]
Having looked at the code, what I posted above makes sense.
When using Diff on an input Diff is processed before Trims are applied.
When using Diff on a mix Diff is processed after Trims are applied.
